lib/notifications/client/speaker.rb

```diff
@@ -95,6 +96,22 @@ def post_with_url(url, form_data)
         perform_request!(request)
       end
 
+      ##
+      # @param reference [String] reference of the notification
+      # @param pdf_file [File] PDF file opened for reading
+      # @see #perform_request!
+      def post_precompiled_letter(reference, pdf_file)
+        content = Base64.encode64(pdf_file.read)
```

Comment:
Ruby has a slightly odd interface for base64 – `.encode64` adds a newline every 60 characters. This wouldn't prevent our apps from decoding it, since python will ignore the newlines, but for example python client would return a singe uninterrupted base64 string, so I think it's worth using `.strict_encode64` (https://ruby-doc.org/stdlib-2.1.3/libdoc/base64/rdoc/Base64.html#method-i-encode64) here instead to match that.

Reply:
Thanks - didn't spot that `encode64` was adding new lines